docs/pages/ktable.vue

```diff
@@ -210,8 +232,141 @@
       </DocsShow>
       <!-- eslint-enable -->
 
-    </DocsPageSection>
+      <!--Table with Default Sort-->
+      <h3>Table with Default Sort</h3>
+      <p>
+        This is an example to show how <code>KTable</code> can be used with the <code>defaultSort</code> attribute to sort the table based on a particular column. This is useful if the table needs to be sorted based on any column from the beginning itself (by default). The <code>defaultSort</code> attribute can be used irrespective of the <code>sortable</code> attribute (as <code>sortable</code> is used to configure whether the client has sorting capabilities or not). 
+      </p>
+
+      <p>
+        The <code>defaultSort</code> attribute takes an object with two properties - <code>columnId</code> and <code>direction</code>. The <code>columnId</code> is the unique identifier of the column based on which the table should be sorted. The <code>direction</code> can be either <code>asc</code> or <code>desc</code>.
+      </p>
+
+      <p>
+        To make use of <code>defaultSort</code>, please ensure that the <code>disableBuiltinSorting</code> attribute is not set to <code>true</code>.
+      </p>
+
+      <DocsShowCode language="html">
+        <h4>Sortable Table with Rows Sorted by 'Age' Column</h4>
```

Comment:
For the case consistency, more in another comment

```suggestion
        <h4>Sortable table with rows sorted by 'Age' column</h4>
```